lib/ansible/modules/cloud/hcloud/hcloud_volume_info.py

```diff
@@ -73,6 +73,11 @@
             returned: always
             type: str
             sample: 10
+        linux_device:
+            description: Path to the device that contains the volume.
+            returned: always
+            type: src
+            sample: /dev/disk/by-id/scsi-0HC_Volume_12345
```

Comment:
please add the version added

~~~diff
+             version_added: "2.10"
~~~